index.bs

```diff
@@ -523,6 +523,17 @@ The W3C Team</h3>
 	(generally to other individuals in the Team)
 	for any of their roles described in this document,
 	except <a href="#tag-participation">participation in the TAG</a>.</p>
+	Decisions being described in this document
```

Comment:
s/Decisions being described in this document as being//

It's unnecessary wordiness

Reply:
Fair enough. I'll take it out unless some third voice argues otherwise.